builtins/amp-ad.js

```diff
@@ -283,6 +283,10 @@ export function installAd(win) {
      * @private
      */
     noContentHandler_() {
+      if (!this.placeholder_) {
+        this.element.style.height = 0;
```

Comment:
Nice co-incidence - i was just pinging you about this 

we have a fallback and a placeholder implemented
the md file does not say anything about the fallback 
but the fallback is the only thing that is displayed when there is no ad content 

I am going to update the MD file to reflect usage for fallback